### DISCUSSION TITLE
metric values generated with package differ from those generated with the builtin LightHouse tool

### DISCUSSION THREAD

**Discussion by riffdr** [GoogleChrome/lighthouse#12601] [General] [answered]
Howdy! 

I have a FE application that loads Web Components and I am testing the performance of my project with this tool. I want to report an issue that I have detected when it comes generating timing metrics locally on a Chrome browser installed in your local machine: the values generated locally with this npm package are different from those you get using the built-in chrome browser tool. This is particular to first-meaningful-paint, first-contentful-paint, largest-contentful-paint, and speed-index. Is this its normal behavior, are we meant to throttle up / down anything?

Thank you in advance for your help and comments.

**Comment by riffdr** [comment]
I have uploaded a video clip showing the differences https://youtu.be/rwUN62gVitg thank you again

**Comment by patrickhulce** [comment, marked as answer]
Those runs have different settings. Comparing desktop v. mobile is comparing two different things. See the [CLI options docs](https://github.com/GoogleChrome/lighthouse#cli-options) for how to run desktop (hint: look at the `--preset=desktop` option).

You might also be interested in reading the [variability documentation](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md) about comparing different runs.

**Reply by riffdr** [reply]
that was my mistake, I had taken for granted that desktop was set as the default value for property `preset`

**Reply by riffdr** [reply]
thank you!
